moveit_core/online_signal_smoothing/src/acceleration_parameters.yaml

```diff
@@ -0,0 +1,16 @@
+online_signal_smoothing:
+  update_rate: {
+    type: double,
+    description: "The expected time in seconds between calls to `doSmoothing` method.",
+    read_only: true,
+    validation: {
+      gt<>: 0.0
+    }
+  }
+  move_group_name: {
```

Comment:
I'd call this `planning_group_name`.
(and if so, update the description as well)

Reply:
Renamed